### DISCUSSION TITLE
Can the button be turned into a button with an icon on top and text below through some settings?

### DISCUSSION THREAD

**Discussion by hestrro** [uwerat/qskinny#478] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/8449f4dc-acfc-470e-8714-a3aeb5946956)

**Comment by uwerat** [comment, marked as answer]
Maybe ( didn't try myself ): button->setFlagHint( QskPushButton::Panel | QskAspect::Direction, Qsk::TopToBottom );

**Reply by hestrro** [reply]
![image](https://github.com/user-attachments/assets/dcb269c6-7c2f-4492-a528-20a8d766e1c1)
  setFlagHint(QskPushButton::Panel | QskAspect::Direction,Qsk::TopToBottom);
        setAlignmentHint(QskPushButton::Text,Qt::AlignCenter);
Also add a center
